private/buf/bufgen/internal/bufgenv2/bufgenv2.go

```diff
@@ -166,7 +165,14 @@ func (g *Generator) Generate(
 		)
 	}
 	for _, inputImage := range inputImages {
-		// TODO: modify this image
+		if genConfig.Managed != nil {
```

Comment:
Shouldn't this be gated on if `managed.enabled = true`?

Reply:
Now using `Enabled` to gate image modifying (I kept the field public to stay consistent with the other fields, it will be made private and there is a TODO that will make these private)